third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -313,8 +333,8 @@ class BlockedToMFMA : public mlir::RewritePattern {
     auto oldAcc = dotOp.getOperand(2);
     auto newAcc = convertAndCastTensor(rewriter, oldAcc, mfmaEnc, mfmaAccType);
 
-    // kWidth is a number of consecutive elements per one instruction per one
-    // thread
+    // kWidth is initialized as k_base, which is the number of elements hold by
```

Comment:
seems to be initialized to `-1` ? also lets use camelCase in the comment as well.